uui-components/src/layout/ScrollBars.tsx

```diff
@@ -6,7 +6,7 @@ import { IHasCX, cx, IHasRawProps } from '@epam/uui-core';
 import css from './ScrollBars.module.scss';
 import type { Scrollbars, ScrollbarProps as LibScrollbarProps, positionValues } from 'react-custom-scrollbars-2';
 
-export interface ScrollbarProps extends IHasCX, Omit<LibScrollbarProps, 'ref'>, IHasRawProps<Scrollbars> {
+export interface ScrollbarProps extends IHasCX, Omit<LibScrollbarProps, 'ref'>, IHasRawProps<React.HTMLAttributes<Scrollbars['container']>> {
```

Comment:
I think `rawProps` meant to be scrollbar container props (which is typeof `HTMLDivElement`),

'cause `Scrollbars` is actually a component type, meaning right now `rawProps` accepts every component property and method, for example it will accept render method:
![2023-11-07_20-11-15_1699374975_1999x1374_scrot](https://github.com/epam/UUI/assets/29537473/aa11c029-b60c-4775-ac4e-aa5b11b7ed85)
and at [L74](https://github.com/epam/UUI/pull/1733/files#diff-88877c2e35d928cfefe91b1e85d4d652922d49c90b98c0a290ce65cc9d9dcc3fL74) `rawProps` spreads to scrollbar props.

Reply:
PR RobPethick/react-custom-scrollbars-2#49, bot not sure it will be merged and released, since repository contains stale pending PRs. Will ping the maintainer